### DISCUSSION TITLE
Bookmark content can be scaled or adjusted font size

### DISCUSSION THREAD

**Discussion by cameol** [sumatrapdfreader/sumatrapdf#3612] [Ideas]
The font size of the bookmark area on the left is too small, I hope that the font size of the bookmark area can be adjusted or zoomed

**Comment by GitHubRulesOK** [comment]
It can in recent versions so lower one is 40 **requires a programme restart**

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/c89b3e02-354d-425d-adde-d81aa71e1fd3)

**Comment by cameol** [comment]
I got it now, thanks
